app/MyApp/index.html

```diff
@@ -227,6 +227,8 @@
     <script src='app/views/CourseStepsView.js'></script>
     <script src='app/views/BecomeMemberForm.js'></script>
     <script src='app/views/CoursesStudentsProgress.js'></script>
+    <script src='app/views/CoursesStatistics.js'></script>
```

Comment:
like in https://github.com/open-learning-exchange/BeLL-Apps/commit/bed65a653d15896e9eec9ebdbf3765e58fadbafa every javascript include in `index.html` needs a special comment